apps/settings/js/app_storage.js

```diff
@@ -28,6 +28,12 @@ var AppStorage = (function AppStorage() {
   function attachListeners() {
     _appStorage.addEventListener('change', getSpaceInfo);
     window.addEventListener('localized', getSpaceInfo);
+    window.addEventListener('panelready', function(e){
```

Comment:
Note that we call to `attachListeners` when there is a visibility change and the visibility is true, so there is a chance that the listener will be added multiple times.
Please detach the handler in `detachListeners`.